lib/matplotlib/tests/test_backend_pgf.py

```diff
@@ -83,8 +82,8 @@ def create_figure():
 
 # test compiling a figure to pdf with xelatex
 @needs_xelatex
-@cleanup(style='classic')
-@switch_backend('pgf')
+@pytest.mark.style('default')
```

Comment:
Hi @QuLogic 
This is the place were I noticed that we were changing stylesheet. Can you confirm this is not normal?